src/lib/init.js

```diff
@@ -48,11 +48,20 @@ export function init(configUpdates) {
 			const cmp = new Cmp(store);
 
 			// Expose `processCommand` as the CMP implementation
-			window[CMP_GLOBAL_NAME] = cmp.processCommand;
+			// window[CMP_GLOBAL_NAME] = cmp.processCommand;
```

Comment:
same, leaving this comment in because i want to call this out when doing another upstream merge.